### ISSUE TITLE
config-package.py needs better documentation

### ISSUE THREAD

**Issue by dataflake** [zopefoundation/meta#13]
Trying to run it against a clean fresh clone of `Products.ZSQLMethods` and noticing that it is very particular about the environment it runs in.

It only ran for me after I created a virtual environment **and** activated it to munge `$PATH`. Then I manually installed all those things `config-package.py` seems to assume are there:

- `tox`
- `virtualenv`
- `zope-testrunner`

Then `tox` yells at me:

```
WARNING: test command found but not installed in testenv
  cmd: /XXX/Products.ZSQLMethods/bin/zope-testrunner
  env: /XXX/Products.ZSQLMethods/.tox/py27
Maybe you forgot to specify a dependency? See also the whitelist_externals envconfig setting.
```

And in the end all tests except for `flake8` fail because import paths are apparently wrong:

```
Test-module import failures:

Module: Products.ZSQLMethods.tests.test_SQL

ModuleNotFoundError: No module named 'Products'


Module: Shared.DC.ZRDB.tests.testArgs

ModuleNotFoundError: No module named 'Shared'

etc etc etc
```

Is this even the way this tool is supposed to be used? Are Zope products namespace packages even an intended consumer?

**Comment by icemac** [comment]
`tox` should be in `$PATH`. About `virtualenv` I am not sure.
`zope.testrunner` should be a `test` dependency of the package.

The tool is intended to be used on namespace packages, too.
Sorry, I am currently lacking the time to work on it. I will probably find time to do so on the next Zope sprint gocept hosts in May.

**Comment by dataflake** [comment]
Independent of config-package.py I have changed part of my own package workflow when I discovered `tox -pall` (thanks Michael) which will **not** work if you let the buildout itself install `tox`, e.g. using `zc.recipe.egg`. My workflow now looks like this after cloning and changing into the clonee:

`python3 -m venv .`
`bin/pip install -U pip setuptools zc.buildout tox`
... do development work...

At that point `tox` and `virtualenv` are available.
